### PR TITLE
Sheet 40-50

### PR DESCRIPTION
**Pull request by esirK**
## Description

Adds historical reports for sheets 40-50

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: reports/report_details.py [hunk 1]
```diff
@@ -681,14 +681,8 @@ def add_transnational_to_regions(regions):
...
             'title': 'Stories where issues of gender equality/inequality are raised by region',
             'desc': '',
-            'reports': [],
-            'historical': '42',
-        },
-        '2010': {
-            'name': '42',
-            'title': 'Breakdown of Stories where issues of gender equality/inequality are raised by region',
-            'desc': 'Stories where issues of gender equality/inequality are raised by region',
             'reports': ['global'],
```
Comment (kilemensi) [inline reports/report_details.py:677]: What's going on here? `42` should be there in **2010**.

**Review by kilemensi — COMMENTED** [review]
Dope, dope @esirK 

----------------------------

1. 40, Can't we repeat region names for 2010 and 2015 historical data? Looks a bit off without them
Screenshot from 2021-03-02 15-21-23
2. 41 `%` is missing from **2010** and `N` is missing from **2015** but the data is there in historical data
Screenshot from 2021-03-02 15-21-39

Screenshot from 2021-03-02 15-21-58

File: reports/report_details.py (hunk 1 above)
Comment (esirK) [reply reports/report_details.py:677]: I might have missed that.

**Comment by esirK** [comment]
> 1. 40, Can't we repeat region names ..

yeah, I think I can add that.
>2. 41 % is missing from 2010
 
lemme check that and update accordingly.

File: reports/report_details.py (hunk 1 above)
Comment (esirK) [reply reports/report_details.py:677]: actually, it's not. Sheet 42 looks like it was introduced in 2015.

**Comment by esirK** [comment]
@kilemensi  for >41 % is missing from 2010 and N is missing from 2015 but the data is there in historical data
the problem with this is how different the 2 sheets are. 
The 2010 sheet doesn't have the yes-no answers that are being used in 2015 and 2020 data.
In 2015, the historical data was only the 2010 data and so plotting it was easily done
Screenshot 2021-03-03 at 08 07 01

in 2020 however, we have 2 historical data that are different to a point where it's not possible to convert one data format to the other. The reason being 2010 data was just about % of females but in 2015 and 2020, it is a `Yes-No` question.

**Comment by kilemensi** [comment]
RE: Sheet `41`, can we check with Jean what does the **2010**  % (and N) stand for @esirK ?

**Comment by esirK** [comment]
> RE: Sheet `41`, can we check with Jean what does the **2010** % (and N) stand for @esirK ?

@kilemensi 
Sarah says we can ignore `2010`.
I'll update the PR to show the 2015 data correctly.

**Review by kilemensi — APPROVED** [review]
:rocket:

**Pull request merged**